### PR TITLE
Completed javascript-local-storage. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image

### CODE REVIEW

File: javascript-local-storage/main.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  $label.setAttribute('class', 'form-check-label flex-grow-1 ml-2');
+  $label.textContent = todo.task;
+
+  $todo.append($formCheck);
+  $formCheck.append($checkbox, $label);
+
+  return $todo;
+}
+
+window.addEventListener('beforeunload', function (event) {
```
Comment (BrettAlbright) [inline javascript-local-storage/main.js]: This should be in the data.js file

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: javascript-local-storage/main.js (hunk 1 above)
Comment (VeraYeh) [reply javascript-local-storage/main.js]: Corrected, thanks.
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**